src/api/application/application-config-default.ts

```diff
@@ -1,34 +1,42 @@
 import { countryLogo } from '@countryconfig/api/application/country-logo'
+import * as fs from 'fs'
+import { join } from 'path'
 
 export const defaultApplicationConfig = {
-  APPLICATION_NAME: 'Farajaland CRS',
+  APPLICATION_NAME: 'SIECM',
+  FIELD_AGENT_AUDIT_LOCATIONS: 'DISTRICT',
+  DECLARATION_AUDIT_LOCATIONS: 'DISTRICT',
+  EXTERNAL_VALIDATION_WORKQUEUE: false,
   BIRTH: {
     REGISTRATION_TARGET: 30,
     LATE_REGISTRATION_TARGET: 365,
     FEE: {
       ON_TIME: 0,
-      LATE: 5.5,
-      DELAYED: 15
+      LATE: 1000.0,
+      DELAYED: 2000
     },
     PRINT_IN_ADVANCE: true
   },
   COUNTRY_LOGO: countryLogo,
   CURRENCY: {
-    languagesAndCountry: ['en-US'],
-    isoCode: 'USD'
+    isoCode: 'MGA',
+    languagesAndCountry: ['en-US'] // mg-MG
   },
   DEATH: {
-    REGISTRATION_TARGET: 45,
+    REGISTRATION_TARGET: 30,
     FEE: {
       ON_TIME: 0,
-      DELAYED: 0
+      DELAYED: 2000
     },
     PRINT_IN_ADVANCE: true
   },
-  PHONE_NUMBER_PATTERN: '^0(7|9)[0-9]{8}$',
+  PHONE_NUMBER_PATTERN: '^[1-9]{1}[0-9]{8}|0[0-9]{9}|[0-9]{13}$',
   NID_NUMBER_PATTERN: '^[0-9]{10}$',
   LOGIN_BACKGROUND: {
-    backgroundColor: '36304E'
+    backgroundImage: `data:image/jpg;base64,${fs
```

Comment:
Could the base64 be replaced with something like this:

```
import { backgroundImage } from '@countryconfig/api/application/login-bg-mdg.jpg';

LOGIN_BACKGROUND: {
  backgroundImage: (`url(${backgroundImage})`),
  imageFit: 'FILL
}
```

Reply:
yup that can be done

Reply:
Generally I think that would be a very good improvement. But after a bit of trying, seems we might have to adjust several places. Best if we had a public endpoint which would only serve this image. However, all this change might be a improvement for general use case but may be a bit large under the Madagascar configuration scope